Earthfile

```diff
@@ -64,7 +64,7 @@ app-build:
   COPY +grpc/proto app/com/lynxanalytics/biggraph/graph_api/proto
   COPY conf conf
   COPY app app
-  COPY built-ins built-ins
+  COPY resources resources
```

Comment:
Could you remind me why did you need to rename the directory?

Reply:
> Could you remind me why did you need to rename the directory?

It's in the PR description!

Reply:
Ah, so now we have a "built-ins" directory inside a "resources" directory?